policyengine_us/reforms/state_dependent_exemptions/repeal_state_dependent_exemptions.py

```diff
@@ -371,6 +371,38 @@ def formula(tax_unit, period, parameters):
                 exempt_status.base[filing_status] + personal_exemptions_added
             )
 
+    class wi_exemption(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Wisconsin exemption"
+        unit = USD
+        definition_period = YEAR
+        reference = (
+            "https://www.revenue.wi.gov/TaxForms2021/2021-Form1f.pdf"
+            "https://www.revenue.wi.gov/TaxForms2021/2021-Form1-Inst.pdf"
+            "https://www.revenue.wi.gov/TaxForms2022/2022-Form1f.pdf"
+            "https://www.revenue.wi.gov/TaxForms2022/2022-Form1-Inst.pdf"
+            "https://docs.legis.wisconsin.gov/misc/lfb/informational_papers/january_2023/0002_individual_income_tax_informational_paper_2.pdf"
+        )
+        defined_for = StateCode.WI
+
+        def formula(tax_unit, period, parameters):
```

Comment:
could you leave a comment on what this is changing and why it can't be done parameterically?

Reply:
The changes are added to readme file, - WI does not separate dependent and head / spouse exemptions - changed `exemptions_count` to `head_spouse_count`

Reply:
Gotcha still think a comment in the code would clarify

Also would refactoring this into two more variables in the original make this cleaner? Can also use adds then